superset-frontend/src/explore/components/AdhocFilterEditPopoverSimpleTabContent.jsx

```diff
@@ -220,7 +242,19 @@ export default class AdhocFilterEditPopoverSimpleTabContent extends React.Compon
     }
   }
 
-  isOperatorRelevant(operator) {
+  isOperatorRelevant(operator, subject) {
+    if (subject && operator && CUSTOM_OPERATORS.includes(operator)) {
+      switch (operator) {
```

Comment:
do we need a switch statement here? seems like it can be an if/then/else since there are only 2 branches.

Reply:
i was thinking maybe in the future we add more custom operators, each of the new operator has some rule to show/hide...

Reply:
I'd say in the future we can refactor then? for now we should only write what's necessary